util/MySqlMigrations/Migrations/20230201192610_LastUserDates.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Bit.MySqlMigrations.Migrations;
+
+public partial class LastUserDates : Migration
+{
+    protected override void Up(MigrationBuilder migrationBuilder)
+    {
+        migrationBuilder.AddColumn<DateTime>(
+            name: "LastEmailChangeDate",
+            table: "User",
+            type: "datetime(6)",
```

Comment:
❓ Any worry about different data providers missing precision? I see MySQL can't support the 7 we use for MSSQL (which is probably overkill and perhaps an improvement in the future).

Reply:
I dont think we have had any issues to date. This is all handled by the EF providers. I think DateTime 7 precision in MSSQL is what is necessary to full qualify a DateTime type in c#. It appears that is is just doing 6 here in MySQL, though, so I guess it would just have a trailing 0 somewhere.